src/stylesheets/elements/_typography.scss

```diff
@@ -55,27 +68,26 @@ button, input, label, optgroup, select, textarea, figcaption {
 }
 
 .btn, button, input {
-  @include fontsize(16);
+  @include fontsize(18);
```

Comment:
Remove `font-size` mixin here and in desktop, replace with `@include epsilon`. Everything else can stay